tools/c7n_salactus/c7n_salactus/cli.py

```diff
@@ -200,6 +201,11 @@ def invoke(f, *args, **kw):
 
     with open(config) as fh:
         data = utils.yaml_load(fh.read())
+
+        # build map of canonical IDs to account name and ID
+        id_map = {account_info['canonical-id']: (account_info['name'], account_info['account-id'])
```

Comment:
Can you set the name and account-id values as a dictionary to avoid bugs later on down the line.